implementations/c/documentation/ockam/demo/handshake/include/handshake_test.h

```diff
@@ -0,0 +1,26 @@
+#include <stdlib.h>
```

Comment:
@robinbudd is the code in `implementations/c/documentation/ockam/demo/` still valid?
I suspect it isn't, we should either update to work with our latest code or remove.